projects/js-packages/base-styles/root-variables.scss

```diff
@@ -73,4 +73,7 @@
   --jp-gap: 16px;
 
   --jp-highlight: #3858e9;
+
+  --automattic-blue-60: #0088BF;
```

Comment:
Instead of adding new colors, I wonder if you'd be able to rely on `@automattic/color-studio` instead? I don't know if the colors you're looking for are available there though.

Reply:
The A4A brand colors don't appear to be available via `@automattic/color-studio`.

Should new colors be added there instead of the `base-styles` package?

Reply:
I'm not sure. Someone from the Design team may know better, I'm not sure how we decide what goes in Color Studio and what doesn't, to be honest.

Reply:
@keoshi could you add context on color studio? I know you've been in that area recently.

Reply:
Hi, friends. We've got a PR up for review that adds all the new A8C colors to Color Studio: https://github.com/Automattic/color-studio/pull/633

Hoping to get this merged today.

Reply:
Pinging @jeffgolenski and @keoshi since they are directly involved in this project.

Reply:
For some strange reason, Filipe's and Noam's replies didn't show up when reviewing this conversation. So, please disregard my late ping.

Reply:
To close the loop on this, once #36751 is merged you should be able to update these values to use Color studio instead.